### PR TITLE
Add Velodrome Finance methodology

### PR DESCRIPTION
**Pull request by SerenityFund**

### CODE REVIEW

File: subgraphs/velodrome-finance/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+The user actions of Velodrome include:
+
+- Deposit and withdraw of liquidity into pools
+- Stake and unstake LP tokens
+- Liquidity provider claim VELO rewards
+- Swap
+- Stake (lock) VELO into veVELO NFT and unstake
+- Stakers vote weekly to decide emission allocation into pools
+- Stakers claim rewards (trading fees, VELO rebase rewards)
+- Stakers claim bribes (a future feature, not bribes are airdropped weekly)
```
Comment (l0c4t0r) [inline subgraphs/velodrome-finance/README.md:35]: typo here? not = now

**Comment by l0c4t0r** [comment]
Looks good, other than the typo that I commented on!

File: subgraphs/velodrome-finance/README.md (hunk 1 above)
Comment (SerenityFund) [reply subgraphs/velodrome-finance/README.md]: thanks amended

**Comment by this-username-is-taken** [comment]
Thanks guys!

**Pull request merged**